src/status_im/contexts/wallet/sheets/account_options/view.cljs

```diff
@@ -112,6 +112,10 @@
   (let [options-height (reagent/atom 0)]
     (fn [{:keys [theme]}]
       (let [accounts               (rf/sub [:wallet/accounts-without-current-viewing-account])
+            view-id                (rf/sub [:view-id])
+            accounts               (if (= view-id :screen/wallet.bridge-select-asset)
```

Comment:
tbh, I think we need to make this solution a bit more generic. 
i.e when you are in any form of send flow you should not be able to switch to a watch only acc.

in the case of send flows we can use something like:

```
(ns status-im.contexts.wallet.common.wizard.utils
  (:require [status-im.contexts.wallet.send.flow-config :as send-flow]))

(defn in-send-flow? [current-screen]
  (some (fn [{:keys [screen-id]}]
          (= current-screen screen-id))
          send-flow/steps))
```

we would need to do the same for bridging but we don't have that flow defined yet, but we could add the basic structure in this pr given that it's just a bunch of screen id's.

Reply:
i.e the use would be 
```
accounts (if (or (wizard-utils/in-send-flow view-id) (wizard-utils/in-bridge-flow view-id))
                         ...
                         ...))
```

Reply:
@J-Son89 switching to a watch-only account in send flow is already not possible right?

Reply:
we'll need to double check that, although I don't think we have blocked that. Wherever the solution for preventing switching to a watch-only account in send flow should be the same for bridging. 
So I am to assume not. 
cc @smohamedjavid

Reply:
and also do we not want to cover the rest of the bridging screens? - it's not just this one, right?

Reply:
@J-Son89 fixed it. Used `:switcher-type  :select-account` on the account switcher of the bridge screens to make it similar to the send flow